README.md

```diff
@@ -102,16 +102,23 @@ The production deployment can also be served as a hapi plugin, located in `serve
 ### Static Site Deployment with Amazon CDK
 This example was created based on the [CDK Static Site Example](https://github.com/aws-samples/aws-cdk-examples/tree/master/typescript/static-site). Below are steps to get setup. IMPORTANT: before following them, update your `cdk.json` file, replacing dummy parameters with real ones for your account and site.
 
+#### Setup your IAM user
+If you have permissions to create an IAM user and add permissions, look for and add these managed permissions:
+```
+cdk-api
+cdk-api-deploy
+```
+
+Otherwise, ask someone who can get you some IAM creds, and use them in these next steps:
 ```bash
 $ npm run build                 # Create a build of the site
 $ sudo pip3 install awscli      # Install AWS CLI locally
 $ aws configure                 # Setup AWS creds, you'll need info from your IAM role
 $ npm install -g aws-cdk        # Install the 'cdk' command
-$ cdk bootstrap
-$ cdk deploy                    # deploy -- this is a long process that will take about 40 minutes
+$ cdk bootstrap -v
```

Comment:
Not everyone rolls with `-v` all the time (it's a debugging tool for me), and it's such a common thing that I don't really think we need to specify it.